lib/PEXv1.ts

```diff
@@ -140,6 +141,12 @@ export class PEXv1 {
    * @return the validation results to reveal what is acceptable/unacceptable about the passed object to be considered a valid presentation definition
    */
   public validateDefinition(presentationDefinitionV1: PresentationDefinitionV1): Validated {
+    let pd = JsonPathUtils.changePropertyNameRecursively(
+      presentationDefinitionV1 as InternalPresentationDefinitionV1,
+      '_const',
+      'const'
+    );
+    pd = JsonPathUtils.changePropertyNameRecursively(pd, '_enum', 'enum');
```

Comment:
Hmzz, we are both passing in the pd object and then returning it. Maybe we shouldn't be returning it, as we are actually modifying the passed in object.

Which does come with another question. Shouldn't we create a deep copy of the passed in object to begin with?

Reply:
updated